### ISSUE TITLE
Pester 5.0.7 Invalid module path error when run in x86 mode

### ISSUE THREAD

**Issue by Arturas-K** [rfennell/vNextBuild#150]
### VSTS Extension you are using
Pester 5.0.7

### Where are you running it?
On-Prem TFS 2015 Update 3 

### Version of Extension/Task
Run Pester Test task

### Expected behavior and actual behaviour
If you select run in x86 option you get following error:
```
Executing the powershell script: c:\Agent\tasks\Pester\5.0.7\Pester.ps1
Re-launching in x86 PowerShell
VERBOSE: Running in x86 PowerShell
Invalid path for ModuleFolder
At C:\Agent\tasks\Pester\5.0.7\Pester.ps1:39 char:17
+ Throw "Invalid path for ModuleFolder"
+ ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+ CategoryInfo : OperationStopped: (Invalid path for ModuleFolder 
:String) [], RuntimeException
+ FullyQualifiedErrorId : Invalid path for ModuleFolder
```
If running with x86 option disabled you get a successful build
```
Executing the powershell script: c:\Agent\tasks\Pester\5.0.7\Pester.ps1
##[debug]Running in AMD64 PowerShell
##[debug]Loading Pester module from [C:\Agent\tasks\Pester\5.0.7\3.4.3]
##[debug]Running Pester from [c:\Agent\Work\26\s\Tests\*.Tests.ps1] output sent to [c:\Agent\Work\26\TestResults\Test-Pester.XML]
Describing Pester validation
[+] Should execute perster test 779ms
Tests completed in 779ms
Passed: 1 Failed: 0 Skipped: 0 Pending: 0 Inconclusive: 
```
As work around I used custom module path in task configuration to point to built in module:**C:\Agent\tasks\Pester\5.0.7\3.4.3**
![pester workaround](https://user-images.githubusercontent.com/16191469/28631865-65dc8c26-7237-11e7-8327-c61501c68b33.PNG)
after this is done you are able to run test in x86 mode:
```
Executing the powershell script: c:\Agent\tasks\Pester\5.0.7\Pester.ps1
Re-launching in x86 PowerShell
VERBOSE: Running in x86 PowerShell
VERBOSE: Loading Pester module from [C:\Agent\tasks\Pester\5.0.7\3.4.3]
VERBOSE: Running Pester from [c:\Agent\Work\27\s\Tests\*.Tests.ps1] output sent
 to [c:\Agent\Work\27\TestResults\Test-Pester.XML]
Describing Pester validation
 [+] Should execute perster test 930ms
Tests completed in 930ms
Passed: 1 Failed: 0 Skipped: 0 Pending: 0 Inconclusive: 0
```

It looks like there is an issue when re-launching

 in x86 mode that resets relative path to built in pester module.

**Comment by ChrisLGardner** [comment]
I can replicate the issue on a hosted build agent in VSTS so I'm working on a fix now.

**Comment by ChrisLGardner** [comment]
Published v5.1 of this extension which should fix the issue.

**Comment by Arturas-K** [comment]
@ChrisLGardner thank you for quick response.
Just verified new version resolved the issue.
:thumbsup:
